doc/using/QuickStartGuide.rst

```diff
@@ -129,14 +129,14 @@ You can analyze this design file, ``ghdl -a adder.vhdl``, and try to execute the
          --  The patterns to apply.
          type pattern_array is array (natural range <>) of pattern_type;
          constant patterns : pattern_array :=
-           (('0', '0', '0', '0', '0'),
-            ('0', '0', '1', '1', '0'),
-            ('0', '1', '0', '1', '0'),
-            ('0', '1', '1', '0', '1'),
-            ('1', '0', '0', '1', '0'),
-            ('1', '0', '1', '0', '1'),
-            ('1', '1', '0', '0', '1'),
-            ('1', '1', '1', '1', '1'));
+            (('0', '0', '0', '0', '0'),
+             ('0', '0', '1', '1', '0'),
```

Comment:
Why has indentation changed?

Reply:
The first example of a code block I found was in the [Quick Start Guide](https://ghdl.readthedocs.io/en/latest/using/QuickStartGuide.html). It used a 3-space indent. Therefore, I standardized all other documents to use a 3-space indent inside code block examples.